reset-aspel-worspace.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Step 1: Reset and clean all submodules
+for dir in packages/*; do
+  if [ -d "$dir/.git" ]; then
+    echo "-----------------------------------------------"
+    echo " 🚧  Resetting and cleaning $dir... Please wait!"
+    echo "-----------------------------------------------"
+
+    # Navigate to the submodule directory
+    cd "$dir"
+
+    # Revert any changes (reset to HEAD)
+    git reset --hard
+
+    # Remove untracked files (including node_modules and package-lock.json)
+    git clean -fdx
+
+    # Go back to the root directory
+    cd - > /dev/null
+  fi
+done
+
+echo "==============================================="
+echo "   ✅  All submodules have been reset and cleaned!  "
+echo "==============================================="
+
+# Step 2: Checkout the main or master branch and pull the latest changes
+echo "-----------------------------------------------"
+echo " 🔄  Checking out main/master and pulling latest changes for all submodules..."
+echo "-----------------------------------------------"
+
+git submodule foreach 'git checkout main || git checkout master'
+git submodule foreach git pull
+
+echo "==============================================="
+echo "   ✅  All submodules have been updated to the latest version!  "
+echo "==============================================="
+
+# Step 3: Remove node_modules and package-lock.json across all submodules
+find packages -type d -name "node_modules" -exec rm -rf {} +
+find packages -type f -name "package-lock.json" -exec rm -f {} +
```

Comment:
My intention was when we finish off working on a feature and it merged to master, now we would want to run the server on master. So in this scenario, the workspace should be in its raw state. As it's a yarn workspace we shouldn't have a separate package-lock.json because it affects the workspace when we install. "I may be wrong, but I believe I faced issues in past"

I agree with you - i.e `.eslintrc` file is manually updated currently. So when we run this script it will revert those changes and a once again we need to update `.eslintrc`.